### PR TITLE
Minor Little Ganga Bugfixes

### PR DESCRIPTION
**Pull request by drmarkwslater**
This PR fixes a couple of things in little ganga:

1. Set the gangadir properly with a username
2. Exports the runMonitoring command to the GPI

### CODE REVIEW

File: python/ganga/__init__.py [hunk 1]
```diff
@@ -56,6 +56,8 @@ def ganga_license():
 interactive = False
 from Ganga.Core.GangaRepository import getRegistrySlice
 Ganga.Core.bootstrap(getRegistrySlice('jobs'), interactive)
+from Ganga.Runtime.GPIexport import exportToInterface
+exportToInterface(ganga, 'runMonitoring', Ganga.Core.monitoring_component.runMonitoring, 'Functions')
```
Comment (rob-c) [inline python/ganga/__init__.py]: You should just be able to pass ```ganga``` as an extra argument to ```bootstrap``` above or you end up exporting this to the ```GPI``` as well which is a little bad as this interface shouldn't be used when doing ```import ganga```

File: python/Ganga/__init__.py [hunk 2]
```diff
@@ -26,6 +26,14 @@ def getLCGRootPath():
     else:
         return ''
 
+def getDefaultUserName():
+    # detect default user (equal to unix user name)
+    import getpass
+    try:
+        return getpass.getuser()
+    except Exception as x:
+        raise Ganga.Utility.Config.ConfigError('Cannot get default user name' + str(x))
```
Comment (milliams) [inline python/Ganga/__init__.py]: If we never handle this exception (and I'm not sure under what circumstances `getuser()` would raise), perhaps we can make the line below be just:
```python
conf_config.addOption('user', getpass.getuser(), ...
```

**Comment by rob-c** [comment]
Is it a trivial fix to get ```import ganga``` to use the .ini files defined in the bash env?
(is it understood what we used to be doing with the old system)

If I try and run with this I suspect unless I tell my ```.gangarc``` that I want to load a plugin it's not done according to any external config it won't pick up things like the Dirac backend for me.

File: python/ganga/__init__.py (hunk 1 above)
Comment (drmarkwslater) [inline python/ganga/__init__.py]: I'm an idiot. Fixed :smile:

File: python/Ganga/__init__.py (hunk 2 above)
Comment (drmarkwslater) [inline python/Ganga/__init__.py]: That makes sense. Fixed.

**Comment by rob-c** [comment]
Go for merge from me, I'm just curious if ```import ganga``` works safely if there is no ```.gangarc``` but I think this is a separate issue for another PR.

**Comment by drmarkwslater** [comment]
I'm not sure and I would prefer to leave that for another PR as I suspect this could be a big(ish) job. I've just created an issue to track it actually (#363). These fixes were what I came across while getting the Tutorial Tests to work as I wanted that to use little ganga.

**Comment by drmarkwslater** [comment]
Assuming everything works, any objections to merge?

**Comment by milliams** [comment]
:+1:

**Pull request merged**